src/test/scala/com/cognite/spark/datasource/TimeSeriesUpsertTest.scala

```diff
@@ -0,0 +1,114 @@
+package com.cognite.spark.datasource
+
+import org.apache.spark.sql.{DataFrame, Row}
+import org.scalatest.FlatSpec
+import org.apache.spark.sql.functions.col
+import com.softwaremill.sttp._
+
+class TimeSeriesUpsertTest extends FlatSpec with SparkTest {
+  val readApiKey = System.getenv("TEST_API_KEY_READ")
+  val writeApiKey = System.getenv("TEST_API_KEY_WRITE")
+
+  it should "successfully both update and insert time series" taggedAs WriteTest in {
+    val initialDescription = """'upsert testing'"""
```

Comment:
Do we need those to be triple quotes? Is the single quote within the string important? Seems like a normal string could do, but maybe we want to test the support for single quotes or something?